x/emissions/proto/emissions/v4/query.proto

```diff
@@ -399,6 +399,34 @@ service QueryService {
     option (cosmos.query.v1.module_query_safe) = true;
     option (google.api.http).get = "/emissions/v4/next_churning_block_by_topic_id/{topic_id}";
   }
+
+  rpc GetCountInfererInclusionsInTopic(GetCountInfererInclusionsInTopicRequest) returns (GetCountInfererInclusionsInTopicResponse) {
```

Comment:
wonder how necessary this was if we're making v5 query.proto?

Reply:
Yes, I think we don't need to add this query on v4 anymore since we are using v5